diff --git a/src/gnuwin32/fixed/etc/Renviron.site b/src/gnuwin32/fixed/etc/Renviron.site
new file mode 100644
index 0000000000..a6d95a7827
--- /dev/null
+++ b/src/gnuwin32/fixed/etc/Renviron.site
@@ -0,0 +1 @@
+R_PLATFORM_PKGTYPE="windows.binary.clang-aarch64"
diff --git a/src/gnuwin32/installer/Makefile b/src/gnuwin32/installer/Makefile
index e7c7dd1c49..d0d886300e 100644
--- a/src/gnuwin32/installer/Makefile
+++ b/src/gnuwin32/installer/Makefile
@@ -31,7 +31,7 @@ DOC_INFO_FILES = AUTHORS COPYING FAQ RESOURCES THANKS CHANGES \
   NEWS NEWS.0 NEWS.1 NEWS.2 
 GW_INFO_FILES = COPYRIGHTS README.Rterm ../../doc/manual/rw-FAQ
 
-ETC_FILES = Rprofile.site Rconsole Rdevga rgb.txt repositories Rcmd_environ
+ETC_FILES = Renviron.site Rprofile.site Rconsole Rdevga rgb.txt repositories Rcmd_environ
 
 
 # we don't want share/{Makefile.in,java,sh} installed
